INSTALL.md

```diff
@@ -11,6 +11,26 @@ PyBitmessage can be run in either one of two ways:
 - from an installed
 package.
 
+## setuptools
+This is now the recommended and in most cases the easiest procedure for installing PyBitmessage. There is also a helper script to resolve dependencies easily.
+
+Go to the directory with PyBitmessage source code and run:
+```
+python checkdeps.py
```

Comment:
`checkdeps.py` is missing `six`, needs to be updated too

Reply:
Added python-six in depends.py

Reply:
checkdeps should be first

Reply:
also mention that checkdeps may have to run repeatedly until it says no more missing dependencies

Reply:
also checkdeps should be updated for ubuntu 20.04